scripts/webpack/webpack.common.js

```diff
@@ -42,7 +42,6 @@ module.exports = {
       https: false,
       string_decoder: false,
     },
-    symlinks: false,
```

Comment:
This is the main fix for watching packages, right?

Reply:
tldr; yes. 

By default files are watched with hashes/timestamps however inside node_modules directory package name and version is used for watching _unless_ it's a symlink which is required for `npm link` or workspaces (outside of yarn PnP) to function.